lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -210,6 +211,12 @@ def invalid_type_error(manifest)
         def missing_keys_validation_error(missing_keys)
           ValidationError.new('manifest_keys.missing', missing_keys: missing_keys.join(', '), count: missing_keys.length)
         end
+
+        def framework_version_iframe_only(package, manifest)
+          if (manifest['frameworkVersion'].nil? || package.iframe_only? && manifest['frameworkVersion'] < '2.0')
```

Comment:
is a string check like this good enough? shouldn't we ideally be doing a string split and major version comparison to guard against stuff like 2.13.0 breaking against < '2.2'